src/App/src/Action/HomePageAction.php

```diff
@@ -26,6 +26,13 @@ public function __construct(Router\RouterInterface $router, Template\TemplateRen
 
     public function __invoke(ServerRequestInterface $request, ResponseInterface $response, callable $next = null)
     {
+        if (! $this->template) {
+            return new JsonResponse([
+                'welcome' => 'Congratulations! You have installed the zend-expressive skeleton application.',
+                'docsUrl' => 'zend-expressive.readthedocs.org',
```

Comment:
Or maybe even `https://docs.zendframework.com/zend-expressive/` as it was previously ..